zkevm-circuits/src/tx_circuit.rs

```diff
@@ -2314,6 +2480,34 @@ impl<F: Field> TxCircuitConfig<F> {
             .map(|(arg, table)| (enable.clone() * arg, table))
             .collect()
         });
+
+        ////////////////////////////////////////////////////////////////////
+        /////////////    4844: Chunk bytes RLC lookups     /////////////////
+        ///////////////// //////////////////////////////////////////////////
+        meta.lookup_any("Keccak table lookup for ChunkHash", |meta| {
+            // Isolate the last row in the fixed section, which belongs to the last tx in the chunk
+            let enable = and::expr(vec![
+                meta.query_fixed(q_enable, Rotation::cur()),
+                not::expr(meta.query_advice(is_calldata, Rotation::cur())),
+                not::expr(meta.query_advice(is_access_list, Rotation::cur())),
+                sum::expr([
+                    meta.query_advice(is_calldata, Rotation::cur()),
+                    meta.query_advice(is_access_list, Rotation::cur()),
```

Comment:
this is:
```python
if not is_calldata and not is_access_list and (is_calldata or is_access_list):
```

Could you check if this is what you meant?

Reply:
This conditional is removed in favor of locating the last row of fixed section with `q_calldata_first`.